lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -20,7 +20,13 @@ def initialize(engine_config:)
         end
 
         def run(file)
-          process_file(file)
+          $stderr.puts("About to analyze #{file}. If analysis fails, consider excluding this file.")
```

Comment:
I would move this into the `else` so we don't show two messages in the skip case.